omnibus/config/software/datadog-agent-finalize.rb

```diff
@@ -116,6 +116,12 @@
             command "echo '# DO NOT REMOVE/MODIFY - used by package removal tasks' > #{install_dir}/embedded/.py_compiled_files.txt"
             command "find #{install_dir}/embedded '(' -name '*.pyc' -o -name '*.pyo' ')' -type f -delete -print >> #{install_dir}/embedded/.py_compiled_files.txt"
 
+            # The prerm and preinst scripts of the package will use this list to detect which files
+            # have been setup by the installer, this way, on removal, we'll be able to delete only files
+            # which have not been created by the package.
+            command "echo '# DO NOT REMOVE/MODIFY - used by package removal tasks' > #{install_dir}/embedded/.installed_by_pkg.txt"
+            command "pushd #{install_dir}; find ./embedded/lib/python*/site-packages >> #{install_dir}/embedded/.installed_by_pkg.txt; popd"
```

Comment:
the wildcard here relies on shell expansion, not sure if we trust it entirely or should rather rely on the semantically identical `find` command only with something like `find . -path './embedded/lib/python*/site-packages/*'`.

Same comment in the preinst/postrm scripts.


Also, I'd use `&&` instead of `;` between the commands, if one fails the `command` method should throw an error.

Reply:
I've tested the find command and it indeed gives the same results without relying in bash expansion.